### PR TITLE
Login signup

### PR DESCRIPTION
**Pull request by AagmanSuri**

### CODE REVIEW

**Comment by shruthipatil11** [comment]
You can follow semantic html. Good looks clean and precise. Good to merge

**Comment by AagmanSuri** [comment]
> You can follow semantic html. Good looks clean and precise. Good to merge

Thanks for the review

File: Css/main.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+.side-bar-items {
+  font-size: 1.3rem;
+  font-weight: 400;
+  margin-bottom: 1rem;
+}
+
+/* .side-bar-flex {
+} */
+
```
Comment (SinchanaMS) [inline Css/main.css:42]: Remove the unused code.
The overall code looks good! 

UI Suggestion: Make the badges on the icons in the header circular. The rest of the UI looks good too! :)

**Review by SinchanaMS — APPROVED** [review]

**Review by keyurshah339 — COMMENTED** [review]
clean code 
go ahead with merge

File: Css/main.css (hunk 1 above)
Comment (AagmanSuri) [reply Css/main.css:42]: Thanks for the suggestions [SinchanaMS](https://github.com/SinchanaMS)

**Comment by AagmanSuri** [comment]
> clean code go ahead with merge

Thanks for the review [keyurshah339](https://github.com/keyurshah339)

**Pull request merged**